tools/tune/common/sof_ucm_blob_write.m

```diff
@@ -13,14 +13,8 @@ function sof_ucm_blob_write(fn, blob8)
 %
 % Copyright (c) 2024, Intel Corporation. All rights reserved.
 
-% Export for UCM cset-tlv with additional 8 bytes header
-SOF_CTRL_CMD_BINARY = 3;
-nh = 8;
-nb = length(blob8);
-ublob8 = zeros(nb + nh, 1, 'uint8');
-ublob8(1:4) = w32b(SOF_CTRL_CMD_BINARY);
-ublob8(5:8) = w32b(nb);
-ublob8(9:end) = blob8;
```

Comment:
Mention this change in commit text. Since the updated sof-ctl adds the required header, adding it in sof_ucm_blob_write() is no more needed.